examples/kernel-dynamic-tile.cpp

```diff
@@ -4,31 +4,33 @@ int main(int RAJA_UNUSED_ARG(argc), char **RAJA_UNUSED_ARG(argv[]))
 {
   std::cout << "\n\nRAJA dynamic_tile example...\n\n";
 
-//Using policy = KernelPolicy<Tile<tile_dynamic<0>, seq_exec, …>>;
-//RAJA::kernel_param<policy>(
-// make_tuple(RangeSegment(0,N)),
-//  make_tuple(32),  // param 0 is referenced by tile_dynamic
-//  [=](int i, int tile_size){
-//
-//  });
+  // Using policy = KernelPolicy<Tile<tile_dynamic<0>, seq_exec, …>>;
+  // RAJA::kernel_param<policy>(
+  //  make_tuple(RangeSegment(0,N)),
+  //   make_tuple(32),  // param 0 is referenced by tile_dynamic
+  //   [=](int i, int tile_size){
+  //
+  //   });
 
   using namespace RAJA;
 
-  kernel_param<
-    KernelPolicy<
-      statement::Tile<1, tile_dynamic<1>, seq_exec,
-        statement::Tile<0, tile_dynamic<0>, seq_exec,
-          statement::For<1, seq_exec,
-             statement::For<0, seq_exec, statement::Lambda<0>>
-          >
-        >
-      >
-    >
-  >(make_tuple(RangeSegment{0,25}, RangeSegment{0,25}),
+  kernel_param<KernelPolicy<statement::Tile<
```

Comment:
This is an example of making nested templates hard to read. I think the original format is much easier to read. @artv3 do you agree?

Reply:
Agree